app.rb

```diff
@@ -0,0 +1,119 @@
+require_relative 'person'
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  def list_books
+    puts 'List of books:'
+    Book.all.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  def list_people
+    puts 'List of people:'
+    Person.all.each do |person|
+      if person.respond_to?(:specialization)
+        print '[Teacher] '
+      else
+        print '[Student] '
+      end
+      puts "Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
```

Comment:
- [ ] I noticed this little anomaly, I was able to create a person (either student or teacher) but I couldn't get their list when I entered **2** in the UI. Kindly fix this. 👍 

<img width="498" alt="Untitled" src="https://github.com/yodit93/school_library/assets/103386183/90dcfdbf-a1f3-431f-afb4-75616ea7076f">

Reply:
- [ ] [optional] @yodit93 great work fixing the changes requested, I would recommend you remove the welcome message as the user is already using the app

![image](https://github.com/yodit93/school_library/assets/61408860/2944a117-6588-433d-a495-c44863846d9b)